tests/baselines/reference/user/jimp.log

```diff
@@ -0,0 +1,8 @@
+Exit Code: 1
+Standard output:
+node_modules/@jimp/core/types/utils.d.ts(16,3): error TS2536: Type '"class"' cannot be used to index type 'T'.
```

Comment:
this is different!

Reply:
This is from the "make T extends any -> T extends unknown" change. `jimp` was making use of a `T extends any` constraint to silence a very blatant error. The types can be rewritten to prevent the error with an `Extract` conditional usage.